common/workunit/workunit.ipp

```diff
@@ -87,6 +86,20 @@ public:
     virtual bool matches(const IStatisticsFilter * filter) const;
 };
 
+class CLocalWUFieldUsageFile : public CInterface, implements IConstWUFieldUsageFile
+{
+    Owned<IPropertyTree> p;
+public:
+    IMPLEMENT_IINTERFACE;
+    CLocalWUFieldUsageFile(IPropertyTree& _p) { p.setown(&_p); }
+
+    virtual IStringVal & getName(IStringVal & ret) const { ret.set(p->queryProp("@name")); return ret; }
```

Comment:
No need to add "override" here since these member functions are new and do not override anything.